src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_CHARACTER = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int TRANSACTION_TYPE_INDEX = 0;
+    private static final int TRANSACTION_VALUE_INDEX = 1;
+    private static final int SUPPLY_AMOUNT_INDEX = 0;
+    private static final int BUY_AMOUNT_INDEX = 1;
+    private static final int RESULT_AMOUNT_INDEX = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] transactionsValues = readCsvFile(fromFileName);
```

Comment:
Why you get array of int from file? I think it will be better to get in String value